token/core/tms.go

```diff
@@ -38,7 +38,7 @@ type PublicParameters struct {
 // It is responsible for creating token management services for different networks.
 type TMSProvider struct {
 	logger                  logging.Logger
-	configProvider          ConfigProvider
+	configProvider          ConfigService
```

Comment:
I suggest to also rename the variable `configProvider` to `configService`

Reply:
okay